src/test/java/com/exasol/projectkeeper/ProjectKeeperIT.java

```diff
@@ -180,12 +184,13 @@ void testChangesFileGeneration(final boolean released) throws IOException, GitAP
     }
 
     private void setupDemoProjectWithDependencyChange(final boolean released) throws IOException, GitAPIException {
-        final Git git = Git.open(this.projectDir.toFile());
-        writePomWithOneDependency("0.1.0", "0.1.0");
-        commitAndMakeTag(git, "0.1.0");
-        writePomWithOneDependency("0.2.0", "0.2.0");
-        if (released) {
-            commitAndMakeTag(git, "0.2.0");
+        try (final Git git = Git.open(this.projectDir.toFile())) {
```

Comment:
Just curious, why did you wrap this in a try block?

Reply:
It's a java try-with-reources. (see https://www.baeldung.com/java-try-with-resources)
It's an elegant way to close the git after the try block.

Reply:
Ok thank you. I thought as much.